eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -199,30 +208,30 @@ object ChannelCodecs extends Logging {
     }).as[Commitments]
 
   val closingTxProposedCodec: Codec[ClosingTxProposed] = (
-    ("unsignedTx" | txCodec) ::
+    ("unsignedTx" | txWithInputInfoCodec.downcast[ClosingTx]) ::
```

Comment:
Right, I took the path of least resistance, but I'll change that in my rebase.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1728/commits/6e2ac8e63326f2e38380362c29e33a1032e50be8

With the new migration procedure, I think the best way to review the codecs is to diff `ChannelCodecs2.scala` with `ChannelCodecs1.scala`, since what I did was copy the previous codec and then modify it.